lib/caasp.pm

```diff
@@ -81,19 +81,19 @@ sub rpmver {
 
 # Export logs from cluster admin/workers
 sub export_cluster_logs {
-    if (is_caasp 'local') {
-        record_info 'Logs skipped', 'Log export skipped because of LOCAL DEVENV';
-    }
-    else {
-        script_run "journalctl > journal.log", 60;
-        upload_logs "journal.log";
-
-        script_run 'supportconfig -b -B supportconfig', 500;
-        upload_logs '/var/log/nts_supportconfig.tbz';
-
-        upload_logs('/var/log/transactional-update.log', failok => 1);
-        upload_logs('/var/log/YaST2/y2log-1.gz') if get_var 'AUTOYAST';
-    }
+    #    if (is_caasp 'local') {
```

Comment:
Please, remove commented code, we have git log in for history.

Reply:
It doesn't work for me. it seems that updates were not installed and velum doesn't show any node with avail upgrades. Still investigating whats wrong. http://dhcp138.suse.cz/tests/844#step/stack_update/14